<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>235</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>235</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>600</width>
    <height>235</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Microsoft JhengHei UI</family>
    <weight>75</weight>
    <bold>true</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>娛樂用音樂撥放器</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="locale">
   <locale language="Chinese" country="Taiwan"/>
  </property>
  <property name="animated">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="SongTitleLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>421</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>22</pointsize>
     </font>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;歌曲名稱&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="SongAlbumLabel">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>60</y>
      <width>411</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;專輯名稱&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="OpenFileButton">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>30</y>
      <width>80</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>開啟檔案</string>
    </property>
   </widget>
   <widget class="QPushButton" name="PlayControlButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>185</y>
      <width>80</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>播放</string>
    </property>
   </widget>
   <widget class="QLabel" name="TimerLabel">
    <property name="geometry">
     <rect>
      <x>505</x>
      <y>185</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>00:00</string>
    </property>
   </widget>
   <widget class="QLabel" name="LyricLabel0">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>100</y>
      <width>500</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="LyricLabel1">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>125</y>
      <width>500</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="LyricLabel2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>150</y>
      <width>500</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
